build.gradle

```diff
@@ -169,6 +169,9 @@ configure(opentelemetryProjects) {
         mavenLocal()
     }
 
+    sourceCompatibility = 1.8
```

Comment:
Could we use the new-style declarations as documented there? https://docs.gradle.org/current/userguide/java_plugin.html#sec:java-extension ?

```java
java {
    sourceCompatibility = JavaVersion.VERSION_1_8
    targetCompatibility = JavaVersion.VERSION_1_8
}
```

Reply:
Sure